content/pages/contributing.md

```diff
@@ -300,24 +304,24 @@ on this later.
 
 For further information see the [GitHub docs on forking a repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
 
-Remember: at this point you should have a local copy of the repository
-in your current working directory. You can safely make changes to
-any of the contents. Once you are ready to contribute your changes
-back to the Pythia repository you will need to submit a Pull Request
-(PR), described later.
+> Remember: at this point you should have a local copy of the repository
```

Comment:
Same as above

Reply:
Not sure that I understand the issue with using (abusing) block quotes, but how about this syntax:

```{Note}
Repository-specific contributor's
information can always be accessed by navigating your web browser
to the appropriate Project Pythia GitHub repository,
[here](https://github.com/ProjectPythia/pythia-foundations) for
example, and clicking on the file named `CONTRIBUTING.md`
```

It's simpler, and we're using it elsewhere in our books (plus I like the rendered results better)

Reply:
OK. Can't figure out how to escape the backticks in my comment above. You may have to simply hit the "edit" button to see the raw text.

Reply:
I see now that we're already using blockquotes for "Note" blocks in the existing guide. I guess I wasn't feeling quite so pedantic when I reviewed that!

I withdraw my comment. As long as we are using the format consistently through the document, it's fine and looks good in the render.

(There are other places in the Foundations book where blockquotes are used for actual quotes, but those are all clear in context)

Reply:
Well, I went ahead and replaced the blockquotes with the syntax above that cannot be displayed in a comment :-). It's consistent with what we are doing elsewhere.

Reply:
I like this better! Resolved...

Reply:
It's a current frustration of mine that those {Note} directives work beautifully in the JupyterBook, but don't work in other forms of Markdown, including in the notebooks themselves before they get rendered into book form...